tests/integration/schema_registry/test_api.py

```diff
@@ -0,0 +1,257 @@
+from uuid import uuid1
+
+from confluent_kafka.schema_registry import CompatibilityType, SchemaRegistryClient
+from confluent_kafka.schema_registry.config import SchemaRegistryConfig
+
+
+def _subject_name(prefix):
+    return "-".join([prefix, str(uuid1())])
```

Comment:
Instantiating a list to concat three strings seems overly complex. Use string cat or format